app/services/symphony_client.rb

```diff
@@ -73,6 +73,15 @@ def fines(patron_key)
     JSON.parse(response.body)
   end
 
+  def reset_pin(library_id)
+    response = request('/user/patron/resetMyPin', method: :post, json: {
+      login: library_id,
+      resetPinUrl: 'future_change_my_pin_path/<RESET_PIN_TOKEN>' # TODO: Based off of #149
```

Comment:
Neither here nor there, but I'm not sure I understand what useful task Symphony is doing for us when we call `resetMyPin`.. just.. sending an email?

Reply:
Yeah exactly.. sending an email with a token tied to the user (that has a short expire) that can then be used to auth for changing the pin